frontend/main/src/CLIConfigParsing.cpp

```diff
@@ -459,80 +466,55 @@ static void viewport_tile_handler(
-        {guiscale_option, "Set scale of GUI, expects float >= 1.0. e.g. 1.0 => 100%, 2.1 => 210%",
-            cxxopts::value<float>(), guiscale_handler},
-        {privacynote_option, "Show privacy note when taking screenshot, use '=false' to disable",
-            cxxopts::value<bool>(), privacynote_handler},
-        {versionnote_option, "Show version warning when loading a project, use '=false' to disable",
-            cxxopts::value<bool>(), versionnote_handler},
-        {param_option, "Set MegaMol Graph parameter to value: --param param=value",
-            cxxopts::value<std::vector<std::string>>(), param_handler},
-        {remote_head_option, "Start HeadNode server and run Remote_Service test ", cxxopts::value<bool>(),
-            remote_head_handler},
-        {remote_render_option, "Start RenderNode client and run Remote_Service test ", cxxopts::value<bool>(),
-            remote_render_handler},
-        {remote_mpi_option, "Start MPI RenderNode client and run Remote_Service test ", cxxopts::value<bool>(),
-            remote_mpirender_handler},
-        {remote_mpi_broadcast_rank_option, "MPI rank that broadcasts to others, default: 0",
-            cxxopts::value<unsigned int>(), remote_mpirank_handler},
-        {remote_headnode_zmq_target_option, "Address and port where to send state via ZMQ",
-            cxxopts::value<std::string>(), remote_zmqtarget_handler},
-        {remote_rendernode_zmq_source_option, "Address and port where to receive state via ZMQ from",
-            cxxopts::value<std::string>(), remote_zmqsource_handler},
-        {remote_headnode_broadcast_quit_option, "Headnode broadcasts mmQuit to rendernodes on shutdown",
-            cxxopts::value<bool>(), remote_head_broadcast_quit_handler},
-        {remote_headnode_broadcast_project_option,
-            "Headnode broadcasts initial graph state after project loading at startup", cxxopts::value<bool>(),
-            remote_head_broadcast_project_handler},
-        {remote_headnode_connect_at_start_option, "Headnode starts sender thread at startup", cxxopts::value<bool>(),
-            remote_head_connect_at_start_handler},
-        {framebuffer_option, "Size of framebuffer, syntax: --framebuffer WIDTHxHEIGHT", cxxopts::value<std::string>(),
-            framebuffer_handler},
-        {viewport_tile_option,
-            "Geometry of local viewport tile, syntax: --tile x,y:LWIDTHxLHEIGHT:GWIDTHxGHEIGHT"
-            "where x,y is the lower left start pixel of the local tile, "
-            "LWIDTHxLHEIGHT is the local framebuffer resolution, "
-            "GWIDTHxGHEIGHT is the global framebuffer resolution",
-            cxxopts::value<std::string>(), viewport_tile_handler},
-        {help_option, "Print help message", cxxopts::value<bool>(), empty_handler}};
+    {     // config name         option description                                                                 type                                        handler
```

Comment:
This code is goind to conflict with master due to the clang-format automatization. Need to delete `// clang-format off` and `// clang-format on` around this block and let it format to our new standards. Else it is goind to collide with master and other PRs.